faker_test.go

```diff
@@ -2316,3 +2437,18 @@ func TestFakeData_RecursiveType(t *testing.T) {
 		}
 	}
 }
+
+func TestFakeDate_ConcurrentSafe(t *testing.T) {
```

Comment:
this test function may not need output anything, otherwise terminal will full of meaningless text